app/models/user.rb

```diff
@@ -1,11 +1,15 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
   has_many :posts, foreign_key: :author_id
   has_many :comments, foreign_key: :author_id
   has_many :likes, foreign_key: :author_id
 
   validates :name, presence: true
-  validates :photo, presence: true
-  validates :bio, presence: true
+  # validates :photo, presence: true
+  # validates :bio, presence: true
```

Comment:
- [ ] You did great job! App is working almost as expected!

- [ ] But to make it appovable we need to work a little bit. For starting, kindly remove unused commented code.

Reply:
Thanks
I have implemented that